### DISCUSSION TITLE
access your vault without a password

### DISCUSSION THREAD

**Discussion by Verguldebarman** [dani-garcia/vaultwarden#3280] [Ideas]
Authenticate with vault by "Log in with device" option.

I think the latest post from Bitwarden explains best. Would be perfect if this feature is added in Vaultwarden 💪

https://bitwarden.com/blog/access-your-bitwarden-vault-without-a-password/

**Comment by BlackDex** [comment]
This feature isn't available on there Self-Hosted instances, so no way for us to also provide this option for now i think. At least, not right now. There must be a reason they do not have this enabled for there own Self-Hosted environment. But I haven't looked into this my self, enough other bugs and quirks to fix.

**Comment by kennymc-c** [comment]
The restrictions for this seem to be lifted since a few days ago when these PRs were merged. I assume this will be included in the next 2023.6 upstream release
https://github.com/bitwarden/server/pull/2934
https://github.com/bitwarden/clients/pull/5441

**Comment by kennymc-c** [comment]
This feature as now been added to the upstream web vault: https://github.com/bitwarden/clients/releases/tag/web-v2023.7.0

**Comment by kennymc-c** [comment]
Looks like this could be included in the next vaultwarden release: https://github.com/dani-garcia/vaultwarden/pull/3592
